fs/build

```diff
@@ -110,6 +110,23 @@ case "$ROOTFS" in
 		wget https://ci.ubports.com/job/xenial-mainline-edge-rootfs-$ARCH_DEB/lastSuccessfulBuild/artifact/out/ubuntu-touch-xenial-edge-$ARCH_DEB-rootfs.tar.gz
 		tar xfp ubuntu-touch-xenial-edge-$ARCH_DEB-rootfs.tar.gz -C $CADMIUMROOT/tmp/root
 	;;
+	fedora)
+		echo "Installing Fedora Linux"
+		# TODO: stop hardcoding this image
+		FEDORA_XZ="https://download.fedoraproject.org/pub/fedora-secondary/releases/35/Spins/aarch64/images/Fedora-Minimal-35-1.2.aarch64.raw.xz"
```

Comment:
i really wish everyone just agreed on naming convention for 32bit arm architecture.
debian calls it `armhf`
linux kernel calls it `arm`
fedora calls it `armhfp`
voidlinux calls it `armv7l`
this is so annoying.

perhaps add `$ARCH_FED` for time being :/